ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -230,6 +230,9 @@ RUN cd /home/user/.theia/ \
  && yarn \
  && yarn theia build
 
+# Configure language server executable for Theia.
+ENV CPP_CLANGD_COMMAND clangd-6.0
```

Comment:
Nit: `ENV` variables from Dockerfiles don't transpire to Bash shells like Cloud9 IDE's Terminal. If this environment variable is supposed to be set for Janitor users, please print it into `/home/user/.bashrc` [like so](https://github.com/JanitorTechnology/dockerfiles/blob/1a8dc2e04016994d2e6c76549a1ad99e37536de8/ubuntu-dev/ubuntu-dev.dockerfile#L120). (And I guess you can merge this with the Theia install instructions).

Reply:
We don't have to, this just have to be set for Theia (so it can start the LS with correct path), not for any user sessions.